ServerCore/Pages/Events/Standings.cshtml.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace ServerCore.Pages.Events
 {
-    [Authorize(Policy = "IsRegisteredForEvent")]
```

Comment:
I might vote to change the match to go the other direction, more restrictive instead of less. I don't have a particular reason for it, it just seems like defaulting to more restrictive is good practice.